pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -1158,6 +1215,7 @@ def main_loop(self):
                     self.reset_near_home()
                 elif state == TestState.MAKING_BACKUP:
                     self.make_backup()
+                self.run_logrotate()
```

Comment:
what will happen if logrotate does not exist? can we fallback to the old way if logrotate is not found? (related to the other comment about local testing on Mac).

Reply:
We can install it on Mac.